src/lib/divider/divider.scss

```diff
@@ -0,0 +1,66 @@
+.mat-divider {
+  display: block;
+  margin: 0;
+  border-top-width: 1px;
+  border-top-style: solid;
+
+  &.mat-divider-vertical {
+    border-top: 0;
+    border-right-width: 1px;
+    border-right-style: solid;
+    position: static;
```

Comment:
`position: static` and `width: auto` should not be necessary as these are the default values on any custom element.

Reply:
Removed